velox/substrait/tests/VeloxSubstraitRoundTripPlanConverterTest.cpp

```diff
@@ -112,3 +112,41 @@ TEST_F(VeloxSubstraitRoundTripPlanConverterTest, values) {
 
   assertPlanConversion(plan, "SELECT * FROM tmp");
 }
+
+TEST_F(VeloxSubstraitRoundTripPlanConverterTest, aggragateCount) {
```

Comment:
typo: aggragateCount -> aggregateCount 

also, rename to 'count' to match the 'sum' test

Reply:
fix.